src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -483,6 +483,7 @@ public void disablePlugin() {
                         return DiscordSRV.this.getConfig();
                     }
 
+
```

Comment:
What's with this?

Reply:
I had some code there at one point that I removed after I figured out a better place to put it

Reply:
Maybe get rid of it for the sake of keeping the commit history somewhat clean